### PR TITLE
Make user interface use new design

### PR DESCRIPTION
**Pull request by jackwherry**
What we're going for:
driver dashboard

### CODE REVIEW

**Comment by jackwherry** [comment]
A few screenshots of what I have now: image
image
image

File: UI/BatteryInfo.qml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    Text {
+        id: primaryValueUnitLabel
+        x: 125
+        y: 127
+        width: 100
+        height: 100
+        color: "#ffffff"
+        text: "KW"
```
Comment (MCLiii) [inline UI/BatteryInfo.qml]: Let's make the unit W for anything under 1000w and KW for above 1000w

File: UI/SolarInfo.qml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    Text {
+        id: primaryValueUnitLabel
+        x: 125
+        y: 127
+        width: 100
+        height: 100
+        color: "#ffffff"
+        text: "W"
```
Comment (MCLiii) [inline UI/SolarInfo.qml]: Same here let's make the unit switch to kw when value is above 1000w

File: UI/Speed.qml [hunk 3]
```diff
@@ -2,364 +2,192 @@ import QtQuick 2.15
...
-            height: 67
+            id: kw
+            x: 140
+            y: 281
+            width: 150
+            height: 55
             color: "#ffffff"
-            text: qsTr("40")
-            font.pixelSize: 65
+            text: "KW"
```
Comment (MCLiii) [inline UI/Speed.qml]: Same here

**Review by MCLiii — COMMENTED** [review]
Values look good, minor change to power display

**Comment by jackwherry** [comment]
Now it switches between watts and kilowatts as appropriate

**Review by MCLiii — APPROVED** [review]
Great work Jack!!

**Pull request merged**